setup.sh

```diff
@@ -0,0 +1,80 @@
+# Colors 
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m'
+
+_install() {
+	if [ ! -e /usr/bin/saathi ];
+	then
+		ln -s `readlink -f $0` /usr/bin/saathi
+		if [ ! $? -eq 0 ];
+		then
+			echo "${YELLOW}Re-run as sudo to setup this script${NC}"
+		fi
+		apt-get install lamp-server^ # install the whole goddammed lamp stack 
+	fi
+}
+
+_check() {
+	if [ -f app/config.php ];
+	then
+		echo "${GREEN}Using config `pwd`/app/config.php${NC}"
+	else
+		echo "${RED}Config not Found.. Please copy the config file and make changes.${NC}"
+		exit 1
+	fi
+}
+
+_start () {
+	_check
+
+	php -S localhost:8000 > /dev/null &
```

Comment:
If we remove `> /dev/null &` we won't need stop script at all. Just `ctrl+c` to stop the server. And also it'll show status in the terminal all the time (which is preferred 'personally' :P)